data/json/items/tool/stationary.json

```diff
@@ -13,6 +13,20 @@
     "symbol": ";",
     "color": "light_gray"
   },
+  {
+    "id": "office_holepunch",
```

Comment:
You need to add some spawn locations to it.

Reply:
I know. I will.